api-report/container-runtime.api.md

```diff
@@ -115,6 +115,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     createDetachedRootDataStore(pkg: Readonly<string[]>, rootDataStoreId: string): IFluidDataStoreContextDetached;
     createSummary(blobRedirectTable?: Map<string, string>, telemetryContext?: ITelemetryContext): ISummaryTree;
     // (undocumented)
+    get currentReferenceTimestampMs(): number | undefined;
```

Comment:
i don't think we should add this. just pass a function to the blob manager, rather than increase the public surface area of container runtime

Reply:
alternatively you could just expose the delete manager to blob manager, and pull it off of that